packages/plugins/src/dashboard-components/layers/absolute-position-layout.js

```diff
@@ -21,16 +26,35 @@ export default class AbsolutePositioningLayout extends Layer {
     dashboard.subscribe('elementSelect', () => {
       if (this.#selectedElement) {
         this.#addResizeInteraction();
+        this.#addDragInteraction();
+      }
+    });
+
+    window.addEventListener('keydown', (e) => {
+      if(e.key == 'Shift') {
```

Comment:
I think we decided to change this to the `Ctrl` key? It's also javascript best practice to use === instead of == as === does type checking while == is a lot looser. Not a big deal but it's generally a good idea to default to that.